src/Kernel32.Shared/Win32Exception.cs

```diff
@@ -46,7 +46,7 @@ public Win32Exception()
         /// <param name="error">The Win32 error code associated with this exception.</param>
         public Win32Exception(Win32ErrorCode error)
 #if DESKTOP
-            : base((int)error)
+            : this(error, error.GetMessage() ?? $"Unknown error (0x{(int)error:x8})")
```

Comment:
I think we should do #314. It will eliminate our custom init.ps1 script and so far in my experience reduces the odds of build failures in VS.